aliases.go

```diff
@@ -49,41 +49,23 @@ func (client *Client) CreateAlias(input AliasCreateInput) ([]string, error) {
 	return output, HandleErrors(err, m.Payload.Errors)
 }
 
-func (client *Client) DeleteDomainAlias(alias string) error {
```

Comment:
This was added yesterday, removing today. We don't want to propagate this pattern.